Actors/CounterActor/CounterActorApplicationCSharp/CounterActorPkg/Code/entryPoint.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
```

Comment:
Did you change the application folder name on purpose? Appended CSharp

Reply:
I appended CSharp so that whenever we deploy both CSharp and Java applications into LRC clusters there won't be any conflicts.